packages/bindings/src/gov.rs

```diff
@@ -86,6 +85,48 @@ pub enum GovProposal {
         /// all code ideas that should be removed from cache to free space
         code_ids: Vec<u64>,
     },
+    // ClientUpdateProposal is a governance proposal. If it passes, the substitute
+    // client's latest consensus state is copied over to the subject client. The proposal
+    // handler may fail if the subject and the substitute do not match in client and
+    // chain parameters (with exception to latest height, frozen height, and chain-id).
+    ClientUpdate {
+        // the client identifier for the client to be updated if the proposal passes
+        subject_client_id: String,
+        // the substitute client identifier for the client standing in for the subject
+        // client
+        substitute_client_id: String,
+    },
+    // UpgradeProposal is a gov Content type for initiating an IBC breaking
+    // upgrade.
+    Upgrade {
```

Comment:
Shouldn't better be called `IbcUpgrade` or so?

Reply:
I don't know. @alpe  does it? I tried to figure it out but it is not clear to me.